cmd/blockchaincmd/deploy.go

```diff
@@ -126,9 +126,16 @@ redeploy the chain with fresh state. You can deploy the same Blockchain to multi
 so you can take your locally tested Blockchain and deploy it on Fuji or Mainnet.`,
 		RunE:              deployBlockchain,
 		PersistentPostRun: handlePostRun,
-		Args:              cobrautils.ExactArgs(1),
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			requiredArgCount := 1
+			if len(args) != requiredArgCount {
+				_ = cmd.Help() // show full help with flag grouping
+				return utils.ErrWrongArgCount(requiredArgCount, len(args))
```

Comment:
also probably this error

Reply:
same as above

Reply:
I mean move utils.ErrWrongArgCount into cobrautils.ErrWrongArgCount

Reply:
this is done in https://github.com/ava-labs/avalanche-cli/pull/2803